usr/lib/soft_stdll/soft_specific.c

```diff
@@ -530,6 +532,19 @@ CK_RV token_specific_aes_key_gen(STDLL_TokData_t *tokdata, CK_BYTE **key,
     return rng_generate(tokdata, *key, keysize);
 }
 
+CK_RV token_specific_aes_xts_key_gen(STDLL_TokData_t *tokdata, CK_BYTE **key,
+                                    CK_ULONG *len, CK_ULONG keysize,
+                                    CK_BBOOL *is_opaque)
+{
+    *key = malloc(keysize);
+    if (*key == NULL)
+        return CKR_HOST_MEMORY;
+    *len = keysize;
```

Comment:
I guess the keysize was already checked before calling this routine? (only 128 and 256)

Reply:
yes, that has already been checked.